### PR TITLE
Kernel tests are not working

### PR DESCRIPTION
**Pull request by mariano-dagostino**
#98

### CODE REVIEW

**Comment by amitaibu** [comment]
Is there a reason for Kernel tests on top of `ExistingSiteTest`?

**Comment by amitaibu** [comment]
Ignore above comment, continuing discussion in #98

File: .ddev/commands/web/phpunit [hunk 1]
```diff
@@ -4,4 +4,4 @@
 ## Usage: phpunit
 ## Example: "ddev phpunit" or "ddev phpunit web/modules/custom/server_general".
 
-cd .. && ./vendor/bin/phpunit -c phpunit.xml.dist  $@
+cd .. && ./vendor/bin/phpunit $@
```
Comment (mariano-dagostino) [inline .ddev/commands/web/phpunit:7]: @amitaibu Not any clue why this is required, but passing a config file make the runner fail:

```
There was 1 error:

1) Drupal\Tests\server_general\Kernel\ServerGeneralTest::testDummy
PHPUnit\Framework\Exception: PHP Fatal error:  Uncaught PHPUnit\Framework\Exception: Could not read "phpunit.xml.dist".
```

File: composer.json [hunk 2]
```diff
@@ -37,6 +37,7 @@
         "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
         "drupal/coder": "^8.3.1",
         "henrikbjorn/lurker": "1.*",
+        "mikey179/vfsstream": "^1.6",
```
Comment (mariano-dagostino) [inline composer.json:40]: @amitaibu Even if this is a dependency of drush, I'm not really sure why is not added to the list of dependencies. Adding without this we get:

```
There was 1 error:

1) Drupal\Tests\server_general\Kernel\ServerGeneralTest::testDummy
Error: Class 'org\bovigo\vfs\vfsStream' not found

```

File: .ddev/commands/web/phpunit (hunk 1 above)
Comment (amitaibu) [reply .ddev/commands/web/phpunit:7]: @mariano-dagostino can you please confirm the file is indeed there. That is, try `ddev ssh`, and try to run it manually, and see if you are able to find the problem

Comment (amitaibu) [reply .ddev/commands/web/phpunit:7]: ok, please adapt PR accordingly - so it includes explicit call to `phpunit.xml.dist`

**Comment by amitaibu** [comment]
Needs a re-roll

**Comment by mariano-dagostino** [comment]
It seems this now works for D9 OOTB #98

**Pull request closed without merging**